tests/system/providers/amazon/aws/example_eventbridge.py

```diff
@@ -45,6 +48,17 @@
 
     # [END howto_operator_eventbridge_put_events]
 
+    # [START howto_operator_eventbridge_put_rule]
+
+    put_rule = EventBridgePutRuleOperator(
+        task_id="put_rule_task",
+        name="Example Rule",
+        event_pattern='{"source": ["example.myapp"]}',
+        description="This rule matches events from example.myapp.",
+    )
+
+    # [END howto_operator_eventbridge_put_rule]
+
 
```

Comment:
Now that you have more than one task in here you will want to add a chain() to set the order they run in.  Check out other system tests for examples